### DISCUSSION TITLE
Support for Futurehome Thermostat with metering?

### DISCUSSION THREAD

**Discussion by mkaut** [Koenkk/zigbee2mqtt#25072] [Devices]
Has anyone tried the new version of [Futurehome Thermostat](https://www.futurehome.io/en_no/shop/thermostat-w), which supports power metering?
Does it get recognized as one of the supported device ([Co020](https://www.zigbee2mqtt.io/devices/Co020.html) or [TS0601](https://www.zigbee2mqtt.io/devices/TS0601_futurehome_thermostat.html)), just without metering, or is not recognized at all?
And in any case, how much work would it be to it supported, incl. metering?
The producer states the device's id as `Co002`, but I have seen a picture of its box in an online shop, which says `Co020`.

(I do not own any, but considering purchasing a couple.)

**Comment by mkaut** [comment]
One more question: I see that the thermostat shows time. I do not see it as an exposed channel and the user manual does not mention anything about setting it manually on the device. Instead, it says: '_Time. This is set automatically after a short time
when connected to the Futurehome Smarthub._' Is this going to work with Z2M as well, or do i risk having a clock that I cannot control?

**Comment by jopppis** [comment]
I do have the model with metering and it does work except for the metering part which does not get recognized by Zigbee2mqtt. Time seems to be synchronized automatically.
